Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1554,6 +1554,7 @@ def lhegensim(fragment,howMuch):
 defaultDataSets={}
 defaultDataSets['2017']='CMSSW_8_1_0_pre5-80X_upgrade2017_realistic_v4_resubUPG17-v'
 defaultDataSets['2023GReco']='CMSSW_8_1_0_pre4-80X_mcRun2_asymptotic_v13_2023LReco-v'
+defaultDataSets['2023LReco']='CMSSW_8_1_0_pre4-80X_mcRun2_asymptotic_v13_2023LReco-v'
 defaultDataSets['2023tilted']='CMSSW_8_1_0_pre4-80X_mcRun2_asymptotic_v13_2023tilted-v'
```

Comment:
so we should just  comment those lines right ? So that they can be used by relvals submitters or simple users (who will need to uncomment locally) - and if they are commented, then the IBs are not broken by any geometry changes

In addition I already asked PdmV to keep this up to date each time relvals are produced ... they are supposed to be in charge..